tests/lib/prepare.sh

```diff
@@ -505,26 +507,18 @@ prepare_all_snap() {
         fi
     done
 
+    echo "Ensure rsync is available"
+    if ! which rsync; then
+        snap install --devmode rsync
```

Comment:
If we rely on the `rsync` snap for tests, can we transfer it to the canonical account? And/or rename it to something less "official" like `rsync-cachio`. I also quickly looked at the snap and it uses version 1.0.0. It probably makes sense to use the version number of the rsync included in the snap.

Reply:
I'll update the name to test-snapd-rsync and transfer that to the canonical account.

Reply:
Thank you, I really appreciate the pattern we use to name snaps that we use to test snapd itself.